src/features/expenses/AddExpense/MainInfo.tsx

```diff
@@ -0,0 +1,142 @@
+import { useCallback, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useLocation, useNavigate, useParams } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+
+import { Input, Select, TextUnderline } from "components";
+import { Column, PageWrapper, Row } from "components/Containers";
+import Header from "components/Header";
+import { ReactComponent as BackIconSVG } from 'assets/images/back-icon.svg';
+import { ReactComponent as ArrowRightSVG } from 'assets/images/arrow-right.svg';
+import { BackgroundFiller, ContentWrapper, CurrencyText, HalfCircleBackground, MainInfoText, MainInfoWrapper, NextStepButtonWrapper } from "./MainInfo.styled";
+import BottomNavigation from "components/BottomNavigation";
+import currencies from "constants/currencies";
+import { ICreateTransaction } from "../types";
+import { newTransactionSelector } from "../slice";
+import { CreateRouteString, ExpenseRouteMode, composeExpenseRoute } from "constants/routes";
+import { saveTransaction } from "../slice";
+
+interface ILocationState {
+  proceed?: boolean;
+}
+
+const MainInfo = () => {
+  const { t } = useTranslation('expenses');
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const location = useLocation();
+  const locationState = location.state as ILocationState;
+
+  const {groupId: groupIdOptional} = useParams();
+  const groupId = groupIdOptional ?? '';
+
+  const transaction = useSelector(newTransactionSelector);
+
+  const [form, setForm] = useState<ICreateTransaction>({
+    name: '',
+    paymentDateUtc: new Date().toISOString().split('T')[0],
+    currencyCode: currencies[0].value,
```

Comment:
Transaction currency should be set by default to group currency

Reply:
Done in: 6a378cecc7209dcac829d4676beef86bb57ef2c2